src/ProjectTemplates/.gitignore

```diff
@@ -4,6 +4,9 @@
 # in source control when people actually create projects from the templates.
 package-lock.json
 
+# Don't track files generated by the build process.
+*/src/**/NuGet.config
+*/src/**/Directory.Build.targets
```

Comment:
❗  This will cause mayhem. Please be more specific.

Reply:
This isn't the repo's global `.gitignore`; these filters apply only to template content. To allow debugging templates locally, the build generates these files and places them directly in the template content folder, but we don't want those files to be checked in.

I've just changed the comment, but removed the new filters anyway because I've pushed an update that reverts the change that uses locally-built MEAI bits.